### ISSUE TITLE
Failure to build on `aarch64`/`armv7`/`riscv64` linux

### ISSUE THREAD

**Issue by sholderbach** [kotauskas/interprocess#63]
**Describe the bug**
Trying to build `interprocess 2.0.0` fails with the targets:
- `aarch64-unknown-linux-gnu`
- `armv7-unknown-linux-gnueabihf`
- `riscv64gc-unknown-linux-gnu`

on this line due to a `u8`/`i8` type mismatch
https://github.com/kotauskas/interprocess/blob/9a1727d7b6d308ab8341d19c929414d546b7fe34/src/os/unix/c_wrappers.rs#L138

```
Error:    --> /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/interprocess-2.0.0/src/os/unix/c_wrappers.rs:138:61
    |
138 |     addr.sun_path[extra..(extra + path.len())].copy_from_slice(path);
    |                                                --------------- ^^^^ expected `&[u8]`, found `&[i8]`
    |                                                |
    |                                                arguments to this method are incorrect
    |
    = note: expected reference `&[u8]`
               found reference `&[i8]`
note: method defined here
   --> /rustc/25ef9e3d85d934b27d9dada2f9dd52b1dc63bb04/library/core/src/slice/mod.rs:3590:12
```

**Additional context**
[Failed nightly build run](https://github.com/nushell/nushell/actions/runs/8946475797/job/24577174493#step:6:923) for [nushell](https://github.com/nushell/nushell)

https://github.com/nushell/nushell/issues/12755

**Comment by kotauskas** [comment]
#62 fixed this, and I've just released that as 2.0.1.

**Comment by sholderbach** [comment]
Thanks for the quick turnaround on this!
